migrations/Version202605011800001488_taoAdvancedSearch.php

```
@@ -0,0 +1,77 @@
+                new IndexMigration(),
+                [
+                    '-i',
+                    $indexName,
+                    '-q',
+                    self::ATTRIBUTES_MAPPING_BODY,
+                ]
+            );
+        }
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)